crates/pallets/starknet/src/blockifier_state_adapter.rs

```diff
@@ -88,7 +88,7 @@ impl<T: Config> State for BlockifierStateAdapter<T> {
 
     fn increment_nonce(&mut self, contract_address: ContractAddress) -> StateResult<()> {
         let current_nonce = Pallet::<T>::nonce(contract_address);
-        let current_nonce: FieldElement = current_nonce.0.into();
+        let current_nonce: FieldElement = Felt252Wrapper::from(current_nonce.0).into();
```

Comment:
Why this change? The previous `From` impl does not work anymore?

Reply:
Yes for some reason `FieldElement` from `StarkFelt` cannot be performed anymore. My fix is to go trough the wrapper.